<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EVSEV2</class>
 <widget class="QWidget" name="EVSEV2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="9" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Lock State</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_lock_state">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>IEC61851 State</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label1">
       <property name="text">
        <string>Time Since State Change</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>LED State</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>GPIOs</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Contactor Check</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>CP PWM Duty Cycle</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Lock Switch</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_iec61851_state">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_adc_values">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Resistances</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Voltages</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_led_state">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_cp_pwm_duty_cycle">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_contactor_check">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QLabel" name="label_energy_meter_error">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>DC Fault Current State</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_gpios">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="label_lock_switch">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Uptime</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="label_uptime">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>ADC Values</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_voltages">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_resistances">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="label_time_since_state_change">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Vehicle State</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_vehicle_state">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Energy Meter Values</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="15" column="1">
      <widget class="QLabel" name="label_energy_meter_values">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Energy Meter Error</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QLabel" name="label_dc_fault_current_state">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Jumper Configuration</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="label_jumper_configuration">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Slot</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="spin_slot_current_2">
       <property name="suffix">
        <string>A</string>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QCheckBox" name="check_slot_active_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Current</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Active</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spin_slot_current_0">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string>A</string>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="check_slot_active_0">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="check_slot_active_1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="check_slot_clear_0">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QLabel" name="label_34">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QLabel" name="label_35">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="0" column="10">
      <widget class="QLabel" name="label_36">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="0" column="11">
      <widget class="QLabel" name="label_37">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="check_slot_clear_1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QCheckBox" name="check_slot_active_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QCheckBox" name="check_slot_clear_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QCheckBox" name="check_slot_active_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QCheckBox" name="check_slot_clear_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QCheckBox" name="check_slot_active_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QCheckBox" name="check_slot_clear_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QCheckBox" name="check_slot_active_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QCheckBox" name="check_slot_clear_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QCheckBox" name="check_slot_clear_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QCheckBox" name="check_slot_active_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="8">
      <widget class="QCheckBox" name="check_slot_clear_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="9">
      <widget class="QCheckBox" name="check_slot_active_8">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="10">
      <widget class="QCheckBox" name="check_slot_active_9">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="11">
      <widget class="QCheckBox" name="check_slot_active_10">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="9">
      <widget class="QCheckBox" name="check_slot_clear_8">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="10">
      <widget class="QCheckBox" name="check_slot_clear_9">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="11">
      <widget class="QCheckBox" name="check_slot_clear_10">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spin_slot_current_1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string>A</string>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QSpinBox" name="spin_slot_current_3">
       <property name="suffix">
        <string>A</string>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QSpinBox" name="spin_slot_current_4">
       <property name="suffix">
        <string>A</string>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QSpinBox" name="spin_slot_current_5">
       <property name="suffix">
        <string>A</string>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QSpinBox" name="spin_slot_current_6">
       <property name="suffix">
        <string>A</string>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="1" column="9">
      <widget class="QSpinBox" name="spin_slot_current_8">
       <property name="suffix">
        <string>A</string>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QSpinBox" name="spin_slot_current_7">
       <property name="suffix">
        <string>A</string>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="1" column="10">
      <widget class="QSpinBox" name="spin_slot_current_9">
       <property name="suffix">
        <string>A</string>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="1" column="11">
      <widget class="QSpinBox" name="spin_slot_current_10">
       <property name="suffix">
        <string>A</string>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="0" column="12">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_9">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="5" column="4">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Duration:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>GPIO Input:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QComboBox" name="combo_gpio_input"/>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QPushButton" name="button_energy_meter_reset">
       <property name="text">
        <string>Reset Energy Meter</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QPushButton" name="button_dc_fault_reset">
       <property name="text">
        <string>Reset DC Fault Current</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>GPIO Output:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="5">
      <widget class="QComboBox" name="combo_gpio_output">
       <item>
        <property name="text">
         <string>Low</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>High</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="3" rowspan="3">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="4" colspan="2">
      <widget class="QPushButton" name="button_energy_meter_detailed">
       <property name="text">
        <string>Show Detailed Energy Meter Info</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Button Config:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QSpinBox" name="spinbox_page">
       <property name="suffix">
        <string> Page</string>
       </property>
       <property name="maximum">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Indication:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QComboBox" name="combobox_indication">
       <item>
        <property name="text">
         <string>Deactivated</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Off</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>75%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>On</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ack</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>nack</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>nag</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="4" colspan="2">
      <widget class="QPushButton" name="button_set_indicator">
       <property name="text">
        <string>Set Indicator LED</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="button_read_page">
       <property name="text">
        <string>Read Page</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QComboBox" name="combo_button_config">
       <item>
        <property name="text">
         <string>Deactivated</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Start Charging</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Stop Charging</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Start And Stop Charging</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0" colspan="6">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="3" rowspan="4">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QSpinBox" name="spin_duration">
       <property name="suffix">
        <string>ms</string>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>Shutdown Input:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="combo_shutdown_input">
       <item>
        <property name="text">
         <string>Ignored</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>On Close</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>On Open</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
